artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
```

Comment:
Very good until first `,`, the last part is almost ok. Correct would be

    , weil wir in diesen Sommer eine (so) große Hitzewelle in Deutschland hatten.

I think it is better here to stay with simple past `hatten` (had) instead of `gehabt haben` (have had), but I'm not sure.